secrets-provider/templates/_helpers.tpl

```diff
@@ -0,0 +1,21 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
+{{- end }}
+{{- end }}
+
+{{/*
+Return the most recent Workload API available
```

Comment:
This is dedicated to Job, so be explicit in the comment and in the name of the function

Reply:
This isn't dedicated to Job. This is for deployment and Job.
If Deployment it will take apps/v1, if Job, it will take batch/v1

Reply:
I am deleted this because we don't yet need this. this would be needed for rotation to switch back and forth